app/src/main/java/org/p2p/wallet/history/ui/history/HistoryFragment.kt

```diff
@@ -27,9 +30,15 @@ class HistoryFragment :
 
     override val presenter: HistoryContract.Presenter by inject()
     private val binding: FragmentHistoryBinding by viewBinding()
+
+    private var isRefreshing = false
     private val adapter: HistoryAdapter by unsafeLazy {
         HistoryAdapter(
-            onTransactionClicked = presenter::onItemClicked,
+            onTransactionClicked = {
+                if (!isRefreshing) {
+                    presenter.onItemClicked(it)
```

Comment:
we can specify for presenter that we clicked on transaction, but not on 'item' in general. Item is more of adapter layer terminology, on presenter level we know we clicked on transaction